framework/CHANGELOG.md

```diff
@@ -7,6 +7,7 @@ Yii Framework 2 Change Log
 - Bug #6363, #8301, #8582, #9566: Fixed data methods and PJAX issues when used together (derekisbusy)
 - Bug #6876: Fixed RBAC migration MSSQL cascade problem (thejahweh)
 - Bug #7627: Fixed `yii\widgets\ActiveField` to handle inputs validation with changed ID properly (dynasource, cebe)
+- Bug #7670: Fixed `yii\web\Request`: forces redirect when unresolved url matches an url with suffix (dynasource)
```

Comment:
Dived into it. 
I started with a fresh new Advanced project with the following setting:

        'urlManager'=>[
            'class'=>'yii\web\UrlManager',
            'enablePrettyUrl' => true,            
        ]

Both following urls render correctly:
- http://localhost/dmn/yii-application/frontend/web/index.php/site/about
- http://localhost/dmn/yii-application/frontend/web/index.php/site/about/////

Seems like unwanted behavior.